robot_ws/src/voice_interaction_robot/launch/deploy_voice_interaction.launch.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+
+import launch
+import launch_ros.actions
+from ament_index_python.packages import get_package_share_directory
+
+
+def generate_launch_description():
+    ld = launch.LaunchDescription([
+        launch.actions.IncludeLaunchDescription(
+            launch.launch_description_sources.PythonLaunchDescriptionSource(
+                os.path.join(get_package_share_directory(
```

Comment:
Might as well use os.path.join to build the entire path:
`os.path.join(get_package_share_directory('turtlebot3_bringup'), 'launch', 'turtlebot3_robot.launch.py')`

(Occurs in multiple places)